services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -54,6 +54,10 @@ export default class ApplyPage extends React.Component<Props> {
 
   render() {
     const { commissions } = this.props;
+    const commissionsWithoutOpenSeats = commissions.filter(
+      commission => commission.openSeats
```

Comment:
This is a good start, but double-check your logic in this filter function. `commission.openSeats` is going to be true-ish when it's not 0.